src/flows_src.rs

```diff
@@ -122,9 +122,7 @@ impl StartRelocateSrc {
     fn try_rpc(&self, rpc: Rpc) -> Option<Self> {
         match rpc {
             Rpc::RefuseCandidate(candidate) => Some(self.vote_parsec_refuse_candidate(candidate)),
-            Rpc::RelocateResponse(candidate, section) => {
-                Some(self.vote_parsec_relocation_response(candidate, section))
-            }
+            Rpc::RelocateResponse(info) => Some(self.vote_parsec_relocation_response(info)),
```

Comment:
Should these RPCs be renamed to match the flow?

Reply:
definitely, but I do not think in this PR as it impact all flows (more than I already had too)

Reply:
No problem, doing it in a follow up PR is fine with me